src/resolvers/deck/createDeck.ts

```diff
@@ -28,3 +39,193 @@ export const createDeck = mutationWithClientMutationId({
     return { deck }
   },
 })
+
+export const importDeck = mutationWithClientMutationId({
```

Comment:
can we rename this? "importing a deck" feels to me like a feature where you upload some sort of file (like a csv, or anki compacted deck) and it creates a deck for you, rather than "cloning" something from the martketplace. Maybe "install" would be a better term here, what do you think?

Reply:
I believe that maybe duplicate or clone could also work.

I changed to install for the moment